lms/envs/common.py

```diff
@@ -2951,3 +2951,13 @@
 # The cache is cleared when Redirect models are saved/deleted
 REDIRECT_CACHE_TIMEOUT = None  # The length of time we cache Redirect model data
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
+
+############## Settings for Neo4j ############################
+NEO4J_CONFIG = {
+    'bolt': True,
+    'password': 'password',
```

Comment:
I think this is the only password in the entire file.  Even if these are dev-only settings, do we have a better way to set up credentials?  Seems like it would be too easy to accidentally clobber the override for prod and use these published values.